modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/search/test/DDMFormInstanceRecordSearchTest.java

```diff
@@ -273,11 +271,41 @@ protected DDMFormFieldValue createLocalizedDDMFormFieldValue(
 			name, localizedValue);
 	}
 
+	private void _addDDMFormInstanceRecord(
+			DDMFormInstance ddmFormInstance, String description, String name,
+			Locale locale)
+		throws Exception {
+
+		DDMForm ddmForm = ddmFormInstance.getDDMForm();
+
+		DDMFormValues ddmFormValues = DDMFormValuesTestUtil.createDDMFormValues(
+			ddmForm, DDMFormValuesTestUtil.createAvailableLocales(locale),
+			ddmForm.getDefaultLocale());
+
+		ddmFormValues.addDDMFormFieldValue(
+			createLocalizedDDMFormFieldValue(
+				"name",
+				HashMapBuilder.put(
+					locale, name
+				).build()));
+
+		ddmFormValues.addDDMFormFieldValue(
+			createLocalizedDDMFormFieldValue(
+				"description",
+				HashMapBuilder.put(
+					locale, description
+				).build()));
```

Comment:
@G-Sa you can remove the empty line and reorder by field name `description` and `name`